src/software/simulation/simulator_robot.cpp

```diff
@@ -349,6 +375,20 @@ void SimulatorRobot::onDribblerBallContact(PhysicsRobot *physics_robot,
 void SimulatorRobot::onDribblerBallStartContact(PhysicsRobot *physics_robot,
                                                 PhysicsBall *physics_ball)
 {
+    // Damp the ball when it collides with the dribbler. We damp each component
+    // of the ball's momentum separately so we have the flexibility to tune this
+    // behavior to match real life.
+    Vector ball_momentum       = physics_ball->momentum();
+    Vector robot_facing_vector = Vector::createFromAngle(physics_robot->orientation());
+    Vector robot_perp_vector   = robot_facing_vector.perpendicular();
```

Comment:
does this account for a moving robot?

Reply:
No, but I think this is a sufficient approximation

Reply:
Damping should use the relative velocity between the ball and the robot. (Arguably, perfect damping is a sufficient approximation, so if we're going to do damping, we should do it right)

Reply:
Perfect damping isn't a perfect approximation (at least to real-life). If it was perfectly damped, by the time the chicker activates there'd be no remaining momentum to "conserve", so we'd always kick at exactly the desired speed. That would be ideal, but not "realistic" which is why we have this medium damping.

Reply:
Will add robot velocity, standby

Reply:
Added robot velocity as well and added tests. I agree this is better.